library/zbd/zbdClient.ts

```diff
@@ -55,20 +49,23 @@ export async function isSupportedRegion(ipAddress: string): Promise<boolean> {
     });
 }
 
-export async function sendKeysend(
-  request: SendKeysendRequest
-): Promise<ZBDSendKeysendPaymentResponse> {
-  log.debug(request);
-  const { data } = await client
-    .post(`https://api.zebedee.io/v0/keysend-payment`, {
-      callbackUrl: `${accountingCallbackUrl}/keysend}`,
-      ...request,
+export async function sendKeysend(request: SendKeysendRequest) {
+  return client
+    .post<ZBDSendKeysendPaymentResponse>(
+      `https://api.zebedee.io/v0/keysend-payment`,
+      {
+        callbackUrl: `${accountingCallbackUrl}/keysend}`,
```

Comment:
I think this is the correct route
```suggestion
        callbackUrl: `${accountingCallbackUrl}/send/keysend}`,
```